frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -0,0 +1,150 @@
+{
+  "Disable": "Disable",
+  "Enable": "Enable",
+  "Name": "Name",
+  "Publisher": "Publisher",
+  "Packages": "Packages",
+  "Package not found": "Package not found",
+  "Cannot create a Subscription to a non-existent package.": "Cannot create a Subscription to a non-existent package.",
+  "Status": "Status",
+  "Availability": "Availability",
+  "Endpoint": "Endpoint",
+  "Registry poll interval": "Registry poll interval",
+  "# of Operators": "# of Operators",
+  "Disabled": "Disabled",
+  "Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.": "Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.",
+  "Enable source": "Enable source",
+  "Create {{item}}": "Create {{item}}",
+  "Edit Subscription": "Edit Subscription",
+  "Uninstall Operator": "Uninstall Operator",
```

Comment:
should message and 'Unstall' button in uninstall modal be translated? 
<img width="461" alt="Screen Shot 2020-12-15 at 6 50 23 PM" src="https://user-images.githubusercontent.com/12692381/102205675-7928c200-3f06-11eb-8603-14cbe8f40909.png">

Reply:
This is addressed in #7499